cloud/blockstore/config/server.proto

```diff
@@ -168,6 +168,10 @@ message TServerConfig
 
     // Path to vhost server executable.
     optional string VhostServerPath = 108;
+
+    // Additional vhost server command line arguments
+    // e.g. ["--pid-file", "/tmp/test-vhost-server-pid", "--verbose", "debug"]
+    repeated string VhostServerExtArgs = 109;
```

Comment:
If I understand correctly those args are needed just for the test to pass the port number.
Instead you can just generate wrapper bash script in the test itself. Something like ...
```bash
#!/bin/bash
exec cloud/blockstore/tools/testing/fake-vhost-server/fake-vhost-server --port 1234 $@
```
and pass it as `VhostServerPath` instead

Reply:
VhostServerExtArgs can be useful for run a vhost-server under the breakpad or for debugging purposes (e.g. --verbose).